Functions.Templates/Templates/CosmosDBTrigger-CSharp/CosmosDBTriggerCSharp.cs

```diff
@@ -21,12 +21,12 @@ public static class CosmosDBTriggerCSharp
             databaseName: "DatabaseValue",
             collectionName: "CollectionValue",
             ConnectionStringSetting = "ConnectionValue",
-            LeaseCollectionName = "leases")]IReadOnlyList<Document> input, TraceWriter log)
+            LeaseCollectionName = "leases")]IReadOnlyList<Document> documents, TraceWriter log)

```

Comment:
Can you update the variable name in line 7 
` public static void Run(IReadOnlyList<Document> input, TraceWriter log)`

Reply:
Done